doc/en/cache.rst

```diff
@@ -214,7 +214,7 @@ of the sleep:
     E       assert 42 == 23
 
     test_caching.py:14: AssertionError
-    1 failed in 0.12 seconds
+    1 failed in 5.43 seconds
```

Comment:
how does this number come to be (the examples are updated by regendoc)

Reply:
(aka - good find, but we need to figure how the heck it got wrong in the first place, else it will turn wrong at the next release)

Reply:
thanks for the quick feedback. i have force-pushed this feature branch with updated changes. let me know if force change is disallowed in the review flow.

Reply:
we are fine with force pushes for as long as we are working out the details, but once we are considering merging, they should be coordinated

my personal preference is to do a chain of commits while working on things (since its a nice history documentation of how things shaped up, and then after review coordinating squashing, so everyone has the opportunity to do ones own revising of ones own git history